kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/build/HelmPushMojoTest.java

```diff
@@ -213,13 +213,13 @@ public void execute_withSkip_shouldSkipExecution() throws Exception {
   }
 
   @Test
-  public void canExecute_withSkip_shouldReturnFalse() {
+  public void shouldSkip_withSkip_shouldReturnTrue() {
```

Comment:
Please don't change existing test. Write a new test for your scenario

Reply:
I see these changes are required since you changed method's name. Sorry for the noise.